onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -310,23 +310,6 @@ cublasStatus_t inline CublasGemmStridedBatched(
       handle, transa, transb, m, n, k, &alpha, A, lda, strideA, B, ldb, strideB, &beta, C, ldc, strideC, batchCount);
 }
 
-struct CublasConfigHelper {
-  cublasPointerMode_t pointer_mode_;
-  cublasMath_t math_mode_;
-  cublasHandle_t cublas_;
-  CublasConfigHelper(cublasHandle_t cublas)
-      : cublas_(cublas) {
-    cublasGetPointerMode(cublas_, &pointer_mode_);
-    cublasGetMathMode(cublas_, &math_mode_);
-    cublasSetPointerMode(cublas_, CUBLAS_POINTER_MODE_HOST);
```

Comment:
CUBLAS_POINTER_MODE_HOST is not handled in CublasMathModeSetter. Does it have impact on performance?